saleor/graphql/attribute/mutations/attribute_value_create.py

```diff
@@ -86,6 +100,7 @@ def perform_mutation(  # type: ignore[override]
     ):
         attribute = cls.get_node_or_error(info, attribute_id, only_type=Attribute)
         instance = models.AttributeValue(attribute=attribute)
+        check_permissions_for_attribute(info.context, attribute, CREATE_PERMISSIONS_MAP)
```

Comment:
Can't you do something like this? :
```python
        attribute_perms = CREATE_PERMISSIONS_MAP[attribute.type]
        cls.check_permissions(context=info.context, permissions=attribute_perms)
```
?
With `cls.check_permission` we will be sure, that the format of error message is the same as the default one.